src/metatrain/utils/data/target_info.py

```diff
@@ -354,12 +376,12 @@ def get_energy_target_info(
         blocks=[block],
     )
 
-    target_info = TargetInfo(
-        quantity="energy",
-        unit=target["unit"],
+    return TargetInfo(
         layout=layout,
+        quantity="energy",
+        unit=target.get("unit", ""),
+        description=target.get("description", ""),
```

Comment:
Here I'm worried about `.get` with a default, for the same reason as above

Reply:
As I wrote above we need at least for description a fallback. I can remove it for the others.